test/6.js

```diff
@@ -0,0 +1,22 @@
+const callX2= require('../solutions/6.js');
+
+const testing = (num)=>{
+  let count = 0; 
+  let passing = true;
+
+  const hello = (a)=>{
+    count+=1;
+    if(a!=count){
```

Comment:
You can shorten your tests by removing the whole block on lines 9-11

`if(a!=count){...`

Then shorten line 14 to check "if the count is equal to num"